main.ts

```diff
@@ -404,6 +414,37 @@ class Translator {
         this.emit('}');
         break;
 
+      case ts.SyntaxKind.ImportDeclaration:
+        var importDecl = <ts.ImportDeclaration>node;
+        this.emit('import');
+
```

Comment:
Nit: I'd rather not put too much vertical space into these blocks here, it's a bit unwieldy already.